include/modem/modem_info.h

```diff
@@ -43,6 +43,13 @@ extern "C" {
 /** Modem firmware version string can be up to 40 characters long. */
 #define MODEM_INFO_FWVER_SIZE 41
 
+// TODO: Confirm - there is no clear guidance in nrf9160 AT command
```

Comment:
Keeping a TODO so I don't forget to discuss with Nordic at some point (note, I'm only merging into my own branch, so this will get eyes again)